class.photon.php

```diff
@@ -388,6 +389,24 @@ public static function filter_the_galleries( $galleries ) {
 		return $galleries;
 	}
 
+
+	/**
+	 * Runs the image widget through photon.
+	 *
+	 * @param array $instance Image widget instance data.
+	 * @return array
+	 */
+	function filter_the_image_widget( $instance ) {
+		if ( Jetpack::is_module_active( 'photon' ) && ! $instance['attachment_id'] && $instance['url'] ) {
+			jetpack_photon_url( $instance['url'], array(
```

Comment:
Yes, it's reasonable. Though that I'm inclined towards the other method since I saw it produced better results, this good too. That said, a filter where you pointed out might be very useful for this and other cases (like for example a user that for whatever reason wants to wrap the image in a particular HTML tag, maybe for styling purposes).